CSharp/ArticleProjects/TaskDependencies/Program.cs

```diff
@@ -21,15 +18,14 @@ public static void Main(string[] args)
             {
                 // Call the asynchronous version of the Main() method. This is done so that we can await various
                 // calls to async methods within the "Main" method of this console application.
-                MainAsync(args).Wait();
+                MainAsync(args).GetAwaiter().GetResult();
             }
-            catch (AggregateException ae)
+            catch (Exception e)
```

Comment:
As this is no longer AggregateException, we should change the "One or more exceptions" language below.